crates/cli-support/src/js.rs

```diff
@@ -765,6 +753,27 @@ impl<'a> Context<'a> {
         "));
     }
 
+    fn expose_constructor_token(&mut self) {
+        if !self.exposed_globals.insert("ConstructorToken") {
+            return;
+        }
+
+        self.globals.push_str("
+            class ConstructorToken {
+                constructor(ptr) {
+                    this.ptr = ptr;
+                }
+            }
+        ");
+
+        self.typescript.push_str("
+            class ConstructorToken {
```

Comment:
I think we probably don't want to expose this in the typescript, right?

in that it's not intended for public consuption

Reply:
I've removed the token and the constructor from the typings, as the constructor needs the token for its signature. I'd argue that in this case we should also remove the `public ptr: number;` from the typings.